charts/model-engine/templates/_helpers.tpl

```diff
@@ -345,6 +345,13 @@ env:
   - name: REDIS_AUTH_TOKEN
     value: {{ .Values.redis.auth }}
   {{- end }}
+  {{- if .Values.redis.authSecret }}
+  - name: REDIS_AUTH_TOKEN
+    valueFrom:
+      secretKeyRef:
+        name: {{ .Values.redis.authSecret }}
+        key: auth_token
+  {{- end }}
```

Comment:
Need refactor/replace above, this should come from secret. Also need to verify this is actually used. Vaguely remember seeing that it was unnecessary

Reply:
Yes let's apply this change